src/services/likeService.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-plusplus */
+
+import { Like, Users } from '../database/models';
+
+const UpdateOrCreate = async(model, data, likeStatus) => {
+    const foundData = await model.findOne({where: {...data, likeStatus}});
+    if(foundData){
+      const item = await model.destroy({where: data});
+      return {item, created: false}
+    }
+   const foundItem = await model.findOne({where: data});
+   if(foundItem){
+     await model.destroy({where: data});
+    }
+     const item = await model.create({...data, likeStatus});
+     return {item, created:true}
+  }
+  const filterLikeArray = async (array)=>{
+    const newArray = []
+    for(let i= 0; i< array.length; i++){
+      newArray.push(array[i].User)
+    }
+    return newArray;
+  }
```

Comment:
add JSON doc on this function so that the team knows why you implemented it since its name is not very descriptive

Reply:
Now there is JS doc for this middleware